Gemfile

```diff
@@ -3,6 +3,9 @@ source 'https://rubygems.org'
 #   spec.add_runtime_dependency '<name>', [<version requirements>]
 gemspec name: 'metasploit-framework'
 
+gem 'packetfu', git: 'https://github.com/adfoster-r7/packetfu', branch: 'add-tests-for-newer-ruby-versions'
```

Comment:
This has been merged now, awaiting a packetfu version bump https://github.com/packetfu/packetfu/issues/207